### DISCUSSION TITLE
stable-diffusion-webui-forge

### DISCUSSION THREAD

**Discussion by fedepalu2** [Mikubill/sd-webui-controlnet#2688] [General]
I would use Deforum in Stable-diffusion-webui-forge but always says "ControlNet not found. Please install", I tried to install from https://github.com/Mikubill/sd-webui-controlnet but can't be installed, may be Webui-forge has a built-in version of controlnet?

**Comment by chaeljc** [comment]
same issue here.  integrated controlnet working ok.  but forge won't allow installation of non integrated controlnet from extensions tab.  keeps unchecking the installed tab.  Then when go to deforum, deforum states '[ControlNet not found. Please install it :)](https://github.com/Mikubill/sd-webui-controlnet)'  :)
